src/liboslexec/opcolor.cpp

```diff
@@ -576,16 +576,25 @@ ColorSystem::set_colorspace (StringParam colorspace)
 
     OSL_HOSTDEVICE void
     ColorSystem::error(StringParam src, StringParam dst, Context sg) {
+#if !defined(__CUDA_ARCH__) || OPTIX_VERSION < 70000
         const char* args[2] = { src.c_str(), dst.c_str() };
         osl_printf (sg,
             (char*) // FIXME!
             "ERROR: Unknown color space transformation \"%s\" -> \"%s\"\n",
             args);
+#else
+        uint64_t fmt_hash = UStringHash::Hash("ERROR: Unknown color space transformation \"%d\" -> \"%d\"\n");
+        uint64_t args[2]  = { 2 * sizeof(int), (uint64_t(dst & 0xffffffff) << 32) + uint64_t(src & 0xffffffff) };
```

Comment:
Just trying to understand this. Seems the args are two strings but you are encoding them in 32 bits? And why do you use 2 * sizeof(int) instead of sizeof(uint64_t)? Don't you wanna pass two 64bit hashes? I guess I'm very confused :)

Reply:
Yeah, I'm not sure what I was thinking.  I think I confused myself over strings being represented as ints.  I'll fix that up.